charts/opentelemetry-operator/templates/clusterrole.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   labels:
     app.kubernetes.io/name: opentelemetry-operator
   name: {{ template "opentelemetry-operator.name" . }}-manager-role
+  namespace: {{ .Release.Namespace }}
```

Comment:
`ClusterRoles` are not a namespaced resource so I believe this will fail

Reply:
@dmitryax great catch. Thanks. Also `ClusterRoleBinding` is not a namespaced resource.